src/translator/byte_array_util.cpp

```diff
@@ -104,5 +104,15 @@ AlignedMemory getShortlistMemoryFromConfig(marian::Ptr<marian::Options> options)
   return loadFileToMemory(shortlist[0], 64);
 }
 
+void getVocabsMemoryFromConfig(marian::Ptr<marian::Options> options,
+                               std::vector<AlignedMemory>& vocabMemorySet){
+  auto vfiles = options->get<std::vector<std::string>>("vocabs");
+  ABORT_IF(vfiles.size() < 2, "Insufficient number of vocabularies.");
```

Comment:
Check should be in loadVocabularies (if mozilla or an API consumer passes only one vocab, we should abort with readable error).

Reply:
I agree on this one.

Reply:
@qianqianzhu Is this resolved?

Reply:
Yes!